### DISCUSSION TITLE
My spotlight is constantly off by what appears to be the height of a static header I have

### DISCUSSION THREAD

**Discussion by paulbarr** [gilbarbara/react-joyride#1086] [General]
Hey. So I take the most simple case and have a single step. 
When I click on the beacon the spotlight always appears below where it should
The width looks correct, the positioning of the height is off
It looks to me to be the height of the header on my page
When I remove the header I still see the issue, but there is code everywhere to deal with the header in my app, so I think somewhere a style is being applied that is causing this issue
As a simple fix, is there a way to pass a static offset that the spotlight can use for positioning?
Then I can counter my static header height
Thanks

In the image below the spotlight is meant to be on the 'Create new +' button
<img width="479" alt="Screenshot 2024-09-16 at 16 53 01" src="https://github.com/user-attachments/assets/eb919eb0-0e71-4355-95d5-5c8091b31238">

**Comment by gilbarbara** [comment]
Hey @paulbarr 
Not exactly. The `scrollOffset` prop would offset the scroll distance and typically work with fixed headers.
However, based on the image, this doesn't seem to be the case.
The spotlight and the tooltip are rendered incorrectly, so this is probably something with your app.

**Reply by paulbarr** [reply]
I found I was setting a margin top somewhere and when I took it out, everything aligns as expected. Thanks
